src/main/java/step2/view/InputView.java

```diff
@@ -0,0 +1,28 @@
+package step2.view;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class InputView {
+
+    private Scanner scanner = new Scanner(System.in);
+    private static final String INPUT_MONEY_MESSAGE = "구입금액을 입력해 주세요.";
+    private static final String INPUT_WINNING_NUMBER_MESSAGE = "지난 주 당첨 번호를 입력해 주세요.";
+
+    public static int inputMoney() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println(INPUT_MONEY_MESSAGE);
+        return scanner.nextInt();
```

Comment:
Scanner가 지역변수에서 생성되네요.
scanner를 상수로 등록해서 사용할 수 있을 것 같습니다.

```suggestion
        System.out.println(INPUT_MONEY_MESSAGE);
        return SCANNER.nextInt();
```

Reply:
반영하겠습니다.